docs/source/cluster_management/clis.rst

```diff
@@ -98,21 +98,30 @@ Read or recalculate the immutable timestamp. Run ``./bin/atlasdb help timestamp`
 migrate
 -------
 
+.. danger::
+
+   Ensure you have backups before attempting a KVS migration.
+   Double check that `migrateConfig` (the target KVS) is specified correctly, since all tables in that KVS will be dropped.
+
 This CLI can help you migrate your AtlasDB client product from one KVS to another.
+The source KVS (which contains the data to migrate) should be specified in the `fromConfig`.
+The target KVS (which we will migrate to) should be specified in the `migrateConfig`.
 You will need to supply two different KVS configurations to the script.
-In the general case you first call ``–-setup``, then ``-–migrate``, then ``-–validate`` each time supplying the old and new configurations.
+In the general case you first call ``--setup``, which creates the necessary tables in the target KVS.
+Then, you call ``--migrate``, which writes the data. If this step fails, it can safely be resumed (with the same arguments).
+The CLI will check and skip past tables that have already been processed.
```

Comment:
I'm not sure if this is meant to be a user manual or one that's more technical in nature, but the semantics of how this is done could be of interest (in particular, that only the most recent versions of data are written).

Reply:
Yeah, some concerns on the other PR are motivated somewhat by making it clear what the pitfalls/limitations of this thing are, which are unfortunately nuanced enough here that it probably requires more depth in the general documentation. That documentation doesn't have to be right here, but we should be documenting this well enough that we understand how this tool works and how to correctly use it.